### ISSUE TITLE
jwk_test.go:609: Errorf call has arguments but no formatting directives

### ISSUE THREAD

**Issue by nim-nim** [square/go-jose#176]
1.10 does not pass unit tests with Go 1.10, and some projects still use it. Please release a gopkg.in/square/go-jose.v1 that is GO 1.10 ready

```
+ GOPATH=/builddir/build/BUILD/go-jose-1.1.0/_build:/usr/share/gocode
+ go test -buildmode pie -compiler gc -ldflags '-extldflags '\''-Wl,-z,relro  '\'''
# gopkg.in/square/go-jose.v1
./jwk_test.go:609: Errorf call has arguments but no formatting directives
./jwk_test.go:615: Errorf call has arguments but no formatting directives
FAIL    gopkg.in/square/go-jose.v1 [build failed]
```

**Comment by csstaub** [comment]
I recommend switching to v2.1.4, which is ready for Go 1.10. That said, I can patch and release a new v1.1.1 that runs on Go 1.10 if you really need it.

**Comment by nim-nim** [comment]
1.1.1 would be really nice, as long as other projects have not switched to v2 :(

**Comment by csstaub** [comment]
Ok, fixed & released in tag v1.1.1

**Comment by nim-nim** [comment]
Thanks a lot !
